internal/acceptance/sql_table_test.go

```diff
@@ -0,0 +1,52 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestAccResourceSqlTable(t *testing.T) {
+	workspaceLevel(t, step{
+		Template: `
+		resource "databricks_schema" "this" {
+			name = "foo"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = "foo"
+			table_type         = "MANAGED"
+			data_source_format = "DELTA"
+			column {
+				name      = "id"
+				position  = 0
+				type_name = "INT"
+				type_text = "int"
+			}
+			column {
+				name      = "name"
+				position  = 1
+				type_name = "STRING"
+				type_text = "varchar(64)"
+			}
+			comment = "this table is managed by terraform"
+		}`,
+	}, step{
+		Template: `
+		resource "databricks_schema" "this" {
+			name = "foo"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
```

Comment:
just to be sure, this means "databricks_sql_table.this table type from previous step will be updated to external"

Reply:
split, and also added a few other acceptance tests for view and updates of various fields